test/Microsoft.Dnx.Testing.CommonUtils/ProjectResolver/IProjectResolver.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+
+using System.Collections.Generic;
+
+namespace Microsoft.Dnx.Testing
+{
+    internal interface IProjectResolver
+    {
+        IEnumerable<string> SearchPaths { get; }
+
+        bool TryResolveProject(string name, out Runtime.Project project);
```

Comment:
Nevermind, Tooling requires this to differentiate against the namespace Microsoft.Dnx.Project.